drivers/SmartThings/zigbee-button/src/zigbee-multi-button/robb/wireless_smarrt_switch_4.lua

```diff
@@ -0,0 +1,73 @@
+local capabilities = require "st.capabilities"
+local constants = require "st.zigbee.constants"
+local clusters = require "st.zigbee.zcl.clusters"
+local device_management = require "st.zigbee.device_management"
+local messages = require "st.zigbee.messages"
+local mgmt_bind_resp = require "st.zigbee.zdo.mgmt_bind_response"
+local mgmt_bind_req = require "st.zigbee.zdo.mgmt_bind_request"
+local log = require "log"
+local zdo_messages = require "st.zigbee.zdo"
+local PowerConfiguration = clusters.PowerConfiguration
+local WindowCovering = clusters.WindowCovering
```

Comment:
Hello, I see that you are referencing WindowCovering, in this line and also a Somfy device below. Was this intentional?